lib/test.gi

```diff
@@ -331,12 +331,23 @@ end);
 ##  <List>
 ##  <Mark>#@local identifierlist</Mark>
 ##  <Item>Run all the tests in the input as if it is in a function with local variable list
-##  <C>identifierlist</C>, which is a comma-separated list of
-##  identifers. Multiple #@local lines may be used.
+##  <C>identifierlist</C>, which is a comma-separated list of identifers, 
+##  terminated with a semicolon.  
```

Comment:
This is still not right: there is no need for a semicolon. In fact, there is not *intention* for supporting a semicolon; it just can be added as a quirk of the current implementation which does not really validate the `identifierlist`.

Reply:
Understood - I was just copying some other files.  New version added.

Reply:
Sorry - don't understand the "Resolve conversation" option - I've just edited the test.gi file.